### DISCUSSION TITLE
Questions regarding disassembly of clay items and some glass items.

### DISCUSSION THREAD

**Discussion by Dracque** [cataclysmbn/Cataclysm-BN#4682] [Q&A] [answered]
Looked up how to recycle clay items since lump of clay is made of "Fresh Clay" while crafted items' material is plain old "Clay". The websites all seemed to agree that once fired, the item could not be recycled into clay and made into new objects.
I'm wondering if I should just ignore that and still add uncrafts for clay items.
If the answer to the above is yes, is there a way to use water as a disassembly tool ?
It would make some sense to re-humidify clay before reusing it.

As for glass uncrafts, there were two odd items in the list: "sealed 3L glass jar" and "sealed glass jar". There doesn't appear to be other sealed containers besides (large) stomachs on the Hitchhiker's Guide.
I don't know if it's normal or not. I was thinking of making uncrafts for those two where instead of glass shards the sealed glass jars would be converted to their non sealed variant.
Any thoughts about it ?
Thanks for taking the time to read.

Edit: The uncrafts for the items "sheet of glass" and "sheet of reinforced glass" don't seem to show up anywhere when doing a research in the code. I'm trying to find the right file to add a disassemble for "sheet of tempered glass" since the two recipes it's in a regular sheet of glass can be used. I don't know wherever the uncrafts for glass sheets are somewhere unusual or if I'm making poor use of GitHub's search function.

**Comment by chaosvolt** [comment, marked as answer]
1. Breaking the fired clay items down into ceramic shards would work, since "ceramic" is a fairly generic term and could mean anything from fired clay to porcelain (likely the majority of shards players will get since that's what sinks, toilets, and the like will be), to born carbide stuff found in ballistic plates. It's not as useful as getting back lumps of clay, but we can definitely improve that by hunting for any additional recipes that use rocks or glass shards that doesn't already allow swapping in ceramic.
2. Sealed jars are just the versions spawned when you encounter unopened jars in store shelves, or in the recipes. They're the ones that preserve contents like cans do, and automatically convert into regular jars that don't perfectly preserve contents once opened. They're unusable in recipes because you're expected to just use the opened versions, since any sealed jars that spawn should currently be occupied holding stuff in them and would automatically be turned into regular jars if you opened them to empty them out.

Worth noting with sealed jars, the behavior is effectively backwards from how tin and aluminum cans work, funny enough. With jars, the usable opened ones are given the "this is fine and normal" item ID while the sealed version is called out as "sealed version," meanwhile the "just a can" item ID is used for the sealed versions and the opened ones are called out separately as the "unsealed" version. XD
